qutebrowser/mainwindow/tabwidget.py

```diff
@@ -480,6 +480,17 @@ def insertTab(self, idx, icon_or_text, text_or_empty=None):
             new_idx = super().insertTab(idx, icon, '')
         self.set_page_title(new_idx, text)
 
+    def wheelEvent(self, event):
+        """Override wheelEvent to make the action configurable."""
+        if config.get('tabs', 'mousewheel-tab-switching'):
+            super().wheelEvent(event)
+        else:
+            tabbed_browser = objreg.get('tabbed-browser', scope='window',
+                                        window=self._win_id)
```

Comment:
What happens here if you send the `wheelEvent` directly to the `tabbed_browser` and let it take care of propagating it to its children (i.e. the focused tab)? If that works, I guess that'd be cleaner.